app/views/components/pageflow/admin/members_tab.rb

```diff
@@ -24,7 +24,7 @@ def build(entry)
             end
           end
         end
-        if authorized? :manage, Pageflow::Entry
+        if authorized? :add_member_to, Pageflow::Entry
```

Comment:
This will always be true since CanCan does not check the block if we pass a class instead of a record.